engine/fixtureParser.js

```diff
@@ -19,6 +19,11 @@ export default class FixtureParser {
 
   readFile(src) {
     const meta = matter.read(src);
+
+    if (Object.keys(meta.data).length === 0) {
+        throw new Error(`Error while parsing '${src}'.`);
```

Comment:
The underlying error is that there's some mistake in the feature file, the fact that the metadata is empty is an effect of that.